### ISSUE TITLE
Bbox query ignores ways with just one node

### ISSUE THREAD

**Issue by mmd-osm** [drolbr/Overpass-API#193]
Currently, bbox queries skip ways with a single node. Although those ways don't make much sense, they technically exist and the query should also return them.

Example:

```C
[out:csv(::id, ::"count:nodes")];
way({{bbox}});
//way(322835317);  // way with single node - not returned by bbox query
foreach(
  out ids;
  node(w);
  out count;
);
```

Link: http://overpass-turbo.eu/s/7l4, http://overpass-turbo.eu/s/8jJ

**Comment by mmd-osm** [comment]
Added a patch [here](https://github.com/mmd-osm/Overpass-API/commit/a176c38fcae7d23ebe461cd4b4c44e8bf04fb307)

**Comment by drolbr** [comment]
Fixed in 07a4e2ab63368ba4182fe1aa449474df7f2df539 by merging patch_issue_193
